source-facebook-marketing/__main__.py

```diff
@@ -0,0 +1,16 @@
+from flow_sdk import shim_airbyte_cdk
+from source_facebook_marketing import SourceFacebookMarketing
+
+shim_airbyte_cdk.CaptureShim(
+	delegate = SourceFacebookMarketing(),
+	oauth2 =
+        {
+            "provider": "facebook",
+            "authUrlTemplate": "https://www.facebook.com/v19.0/dialog/oauth?client_id={{#urlencode}}{{{ client_id }}}{{/urlencode}}&redirect_uri={{#urlencode}}{{{ redirect_uri }}}{{/urlencode}}&state={{#urlencode}}{{{  state }}}{{/urlencode}}&scope=ads_management,ads_read,read_insights,business_management",
```

Comment:
@flavioriper tested and confirmed that he was still able to do the OAuth flow in the dashboard after this change